lbz/jwt_utils.py

```diff
@@ -38,7 +40,7 @@ def validate_jwt_properties(decoded_jwt: dict) -> None:
         raise Unauthorized(f"{issuer} is not an allowed token issuer")
 
 
-def decode_jwt(auth_jwt_token: str) -> dict:  # noqa:C901
+def decode_jwt(token: str) -> dict:  # noqa:C901
```

Comment:
I would stay consistent with the above function and name this argument either `token` or `encoded_token`.

But in both places the same way.